.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -0,0 +1,33 @@
+name: Bug Report
+description: Create a report to help us improve SciPy
+title: "[Bug]: "
+labels: [defect]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to file a bug report.
+  - type: markdown
+    attributes:
+        value: Please fill in the fields below, deleting the sections that don't apply to your issue.
+  - type: markdown
+    attributes:
+        value: You can view the final output by clicking the preview button above.
+  - type: input
+    attributes:
+      label: Reproducing Code Example
+      description: 'If you place your code between the triple backticks below, it will be rendered as a code block.'
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Error message
+      description: 'If any, paste the *full* error message inside a code block as above (starting from line Traceback)'
```

Comment:
Same `render: shell` (assuming that works).  I guess maybe there is `render=python` or even tb as a better default/guess?)

Reply:
Done!